tests/integration/kubernetes/k8s-nvidia-nim.bats

```diff
@@ -14,7 +14,6 @@ export RUNTIME_CLASS_NAME
 export LOCAL_NIM_CACHE="/opt/nim/.cache"
```

Comment:
A comment generic to both  bats files: Some of the changes may have effect on the flow of the non-confidential CI tasks.  Maybe quickly check if we need or want to gate some of the changes behind CC or non-CC (e.g., call to kbs_set_resource_base64)